Makefile

```diff
@@ -81,10 +86,19 @@ localeextract:
 	python manage.py extract
 	python manage.py merge
 
+locale-populate-react:
```

Comment:
It took me a while to figure out what this code is doing. Consider adding a comment here to explain the concatenation and extraction going on with msgcat and msgmerge. 

It would also be useful to explain (because I'm not entirely clear) what the bottom line is here. I'm pretty sure I understand that this means that React components can use any string that is already in use by python or the existing static javascript files, and no new work will be created for translators.  I'm not sure what happens if our React code starts using new strings... Will those end up on Pontoon for translation? Or will they just remain untranslated for now?